example/sdfGeneration.js

```diff
@@ -262,17 +258,14 @@ function updateSDF() {
 					const index = x + y * dim + z * dim * dim;
 					const dist = bvh.closestPointToPoint( point, target ).distance;
 
-					// get the face normal to determine if the distance should be positive or negative
-					const faceIndex = target.faceIndex;
-					const i0 = indexAttr.getX( faceIndex * 3 + 0 );
-					const i1 = indexAttr.getX( faceIndex * 3 + 1 );
-					const i2 = indexAttr.getX( faceIndex * 3 + 2 );
-					tri.setFromAttributeAndIndices( posAttr, i0, i1, i2 );
-					tri.getNormal( normal );
-					delta.subVectors( target.point, point );
+					// raycast inside the mesh to determine if the distance should be positive or negative
+					ray.origin.copy( point );
+					ray.direction.set( 0, 0, 1 );
+					const hit = bvh.raycastFirst( ray, THREE.DoubleSide );
+					let isInside = hit && hit.face.normal.dot( ray.direction ) > 0.0;
```

Comment:
This can be "const" - also "inside" should be denoted as being < 0.

Reply:
Good shout on the `const`, but I think the math is right (the origin is inside when the dot of the ray direction and hit normal is positive):
![image](https://github.com/user-attachments/assets/50f5bdac-3bf3-4eec-b350-182ecc1c4555)

Also it looks funny when I set it to < 0:
![image](https://github.com/user-attachments/assets/55120748-e194-4878-9864-3f3aa6978d77)